components/self_hosted_purchase_modal/index.tsx

```diff
@@ -452,6 +455,29 @@ export default function SelfHostedPurchaseModal(props: Props) {
         />
     );
 
+    const canRetry = state.error !== '422';
+    const resetToken = () => {
+        try {
+            Client4.bootstrapSelfHostedSignup(true).
+                then((data) => {
+                    reduxDispatch({
+                        type: HostedCustomerTypes.RECEIVED_SELF_HOSTED_SIGNUP_PROGRESS,
+                        data: data.progress,
+                    });
+                });
+        } catch (e) {
```

Comment:
[ES2019 introduced Optional Catching binding](https://selvaganesh93.medium.com/javascript-whats-new-in-ecmascript-2019-es2019-es10-35210c6e7f4b). 3/5, and I see the project's target is ES2020 so that you can omit the variable.